packages/wrangler/src/publish.ts

```diff
@@ -161,119 +163,113 @@ export default async function publish(props: Props): Promise<void> {
       : [],
   };
 
-  if (triggers) {
-    console.log("publishing to workers.dev subdomain");
+  const start = Date.now();
+  const formatTime = (duration) => {
+    return `(${(duration / 1000).toFixed(2)} sec)`;
+  };
+
+  const notProd = !props.legacyEnv && props.env;
+  const workerName = notProd ? `${scriptName} (${envName})` : scriptName;
+  const workerUrl = notProd
+    ? `/accounts/${accountId}/workers/services/${scriptName}/environments/${envName}`
```

Comment:
@Electroid wouldn't this make publishing as a service the default?

Reply:
If the standard (non-environment) endpoint is used, it will still publish as a service, the only difference is that it will use the default environment (`production` if none exist).